File: weather-api/src/main/java/com/thewizrd/weather_api/tomorrow/Rootobject.java
package com.thewizrd.weather_api.tomorrow;

import com.squareup.moshi.Json;
import com.squareup.moshi.JsonClass;

@JsonClass(generateAdapter = true, generator = "java")
public class Rootobject {

    @Json(name = "data")
    private Data data;

    //@Json(name = "warnings")
    //private List<WarningsItem> warnings;

    public void setData(Data data) {
        this.data = data;
    }

    public Data getData() {
        return data;
    }

	/*
	public void setWarnings(List<WarningsItem> warnings){
		this.warnings = warnings;
	}

	public List<WarningsItem> getWarnings(){
		return warnings;
	}
	 */
}

File: weather-api/src/main/java/com/thewizrd/weather_api/aqicn/Rootobject.kt
package com.thewizrd.weather_api.aqicn

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Rootobject(
    @Json(name = "data")
    var data: Data? = null,

    @Json(name = "status")
    var status: String? = null
)